src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -216,7 +216,7 @@ export const TrackPanelListItem = (props: TrackPanelListItemProps) => {
         </label>
         <div className={styles.ellipsisHolder}>
           <ImageButton
-            buttonStatus={Status.ACTIVE}
+            buttonStatus={Status.UNSELECTED}
```

Comment:
this is a bit confusing — if the ellipsis button has only a single status, shouldn't it then be default?